### ISSUE TITLE
AC3.2.1: Disabling AC_FENCE causes a build error

### ISSUE THREAD

**Issue by muramura** [ArduPilot/ardupilot#15250]
## Bug report

I want to test the AC 3.2.1 quad with APM.
I want to verify the operation of AC3.2.1 by making it minimally functional.
I have disabled AC_FENCE and I get a build error.
I have a video of the APM quadcopter flying in the world that is still being shown on video sites.
I think eliminating the build error would be good for ArduCopter users around the world.
I will now make some changes to the source.
Where do I advertise it?

**Issue details**

<img width="1440" alt="スクリーンショット 2020-09-06 14 23 23" src="https://user-images.githubusercontent.com/646194/92318818-a5ea5500-f04c-11ea-9ba8-a013eed6d405.png">

**Version**

AC3.2.1

**Platform**
[  ] All
[  ] AntennaTracker
[X] Copter
[  ] Plane
[  ] Rover
[  ] Submarine

**Airframe type**

QUAD

**Hardware type**

APM

**Logs**
_Please provide a link to any relevant logs that show the issue_

**Comment by muramura** [comment]
I would add a change to enable and disable access to fence.pre_arm_check() in the AC_FENCE preprocessor.

Where do I go to promote these changes?

![Screenshot from 2020-09-08 08-02-20](https://user-images.githubusercontent.com/646194/92419771-6cc7f700-f1aa-11ea-9aa8-6310d13d3a1b.png)

**Comment by rmackay9** [comment]
We don't support the AC3.2.1 version anymore though so I don't think will accept any patches to the code or the wiki for this old version.

**Comment by peterbarker** [comment]
@muramura -san  we haven't attempted to run software on this platform for a very, very long time now.

As a development team we do not have the resources or inclination to help out on any tweaks users might want to introduce newer features onto their working machines.

Just asking here has already cost time from several developers to evaluate - I've looked and thought about this one 3 times now, Randy at least once and probably several other devs in passing that didn't respond.

You're obviously welcome to try to backport features, but we've moved forward as a team and we'd prefer not to have to bounce these requests for assistance.
